dm/master/openapi.go

```diff
@@ -88,17 +90,64 @@ func (s *Server) GetDocHTML(ctx echo.Context) error {
 
 // DMAPICreateSource url is:(POST /api/v1/sources).
 func (s *Server) DMAPICreateSource(ctx echo.Context) error {
```

Comment:
will this method print some logs to audit?

Reply:
there is a access log in openapi logger. 
it looks like this: `
[2021/08/30 17:50:27.642 +08:00] [INFO] [logger.go:47] [Success] [component=openapi] [size=128] [host=127.0.0.1:8261] [status=201] [method=POST] [protocol=HTTP/1.1] [remote_ip=127.0.0.1] [user_agent=python-requests/2.22.0] [duration=171.443871ms] [request="POST /api/v1/sources"]`